__tests__/include-dependencies.js

```diff
@@ -322,6 +381,8 @@ test('disables caching by default', t => {
   const instance = createTestInstance();
   const file = path.join(__dirname, 'fixtures', 'thing.js');
   const list1 = instance.getDependencies(file, []);
+  instance.checkedFiles.clear(); // clear would be called when through createDeploymentArtifacts
```

Comment:
This is a bit weird. In reality, `getDependencies` would never be called like this, so the checkedFiles field would always be cleared between calls.